plugin/src/main/java/net/thenextlvl/hologram/implementation/CraftHologramLoader.java

```diff
@@ -64,11 +67,27 @@ private static class ClientHologramLoader extends WeakHashMap<Player, Set<Hologr
 
         private void load(Hologram hologram, CraftPlayer player) {
             addHologram(player, hologram);
-            var display = ((CraftDisplay) hologram).getHandle();
-            player.getHandle().connection.send(display.getAddEntityPacket());
+            CraftDisplay display = (CraftDisplay) hologram;
+            player.getHandle().connection.send(createAddEntityPacket(display));
             update(hologram, player);
         }
 
+        private Packet<?> createAddEntityPacket(CraftDisplay display) {
+            return new ClientboundAddEntityPacket(
+                    display.getEntityId(),
+                    display.getUniqueId(),
+                    display.getLocation().getX(),
+                    display.getLocation().getY(),
+                    display.getLocation().getZ(),
+                    display.getLocation().getPitch(),
+                    display.getLocation().getYaw(),
+                    display.getHandle().getType(),
+                    0,
+                    new Vec3(0, 0, 0),
+                    display.getLocation().getYaw()
```

Comment:
Please use the direct getter instead of `getLocation().getXYZ()`
the `getLocation` method returns a new object every time.